src/services/ui/e2e/tests/home/index.spec.ts

```diff
@@ -1,30 +1,47 @@
+import * as Libs from "../../../../../libs/secrets-manager-lib";
 import { test, expect } from "@playwright/test";
 import { testUsers } from "e2e/utils/users";
+const stage =
+  process.env.STAGE_NAME === "production" || process.env.STAGE_NAME === "val"
+    ? process.env.STAGE_NAME
+    : "default";
+const secretId = `${process.env.PROJECT}/${stage}/bootstrapUsersPassword`;
 
-const password = process.env.BOOTSTRAP_USERS_PW!;
+const password = (await Libs.getSecretsValue(
+  process.env.REGION_A as string,
+  secretId
+)) as string;
```

Comment:
im just wondering why we need to do this. im sure theres a good reason but lmk

Reply:
Yes lets chat about, I want to bounce ideas off of you!